WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -833,6 +810,41 @@ class PostListViewModel @Inject constructor(
         _snackbarAction.postValue(snackbarHolder)
     }
 
+    fun scrollToPost(localPostId: Int) {
+        val data = pagedListData.value
+        if (data != null) {
+            _scrollToPosition.value = findItemListPosition(data, localPostId) ?: run {
+                AppLog.e(AppLog.T.POSTS, "ScrollToPost failed - the post not found.")
+                null
+            }
+        } else {
+            // store the target post id and scroll there when the data is loaded
+            scrollToLocalPostId = localPostId
+        }
+    }
+
+    fun onDataUpdated(data: PagedPostList) {
+        val localPostId = scrollToLocalPostId
+        if (localPostId != null) {
+            scrollToLocalPostId = null
```

Comment:
Should we do this after we find and scroll to the this post? For example, what if the data is updated and this post is not in the list, but it's updated again after and it's in the list?